docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -0,0 +1,309 @@
+"""
+=======================
+Solar System Animation
+=======================
+
+In this tutorial, we will create an animation of the solar system
+using textured spheres. We will also show how to manipulate the
+position of these sphere actors in a timer_callback function
+to simulate orbital motion.
+"""
+
+import numpy as np
+from fury import window, actor, utils, io
+import itertools
+from fury.data.fetcher import read_viz_textures, fetch_viz_textures
+
+##############################################################################
+# Create a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Load in a texture for each of the actors. To do this, we will create
+# a function called ``init_planet``, which will initialize each planet actor
+# given its corresponding filename and actor name. It will also add each
+# actor to the scene that has already been created.
+
+planet_filenames = ["8k_mercury.jpg", "8k_venus_surface.jpg",
+                    "1_earth_8k.jpg", "8k_mars.jpg", "jupiter.jpg",
+                    "8k_saturn.jpg", "8k_saturn_ring_alpha.png",
+                    "2k_uranus.jpg", "2k_neptune.jpg",
+                    "8k_sun.jpg"]
+fetch_viz_textures()
+
+
+def init_planet(filename):
+    """Initialize a planet actor.
+
+    Parameters
+    ----------
+    filename : str
+        The filename for the corresponding planet texture.
+
+    Returns
+    -------
+    planet_actor: actor
+        The corresponding sphere actor with texture applied.
+    """
+    planet_file = read_viz_textures(filename)
+    planet_image = io.load_image(planet_file)
+    planet_actor = actor.texture_on_sphere(planet_image)
+    scene.add(planet_actor)
+    return planet_actor
+
+
+##############################################################################
+# Use the ``map`` function to create actors for each of the texture files
+# in the ``planet_files`` list. Then, assign each actor to its corresponding
+# actor in the list.
+
+planet_iterator = map(init_planet, planet_filenames)
+planet_actor_list = list(planet_iterator)
```

Comment:
One line is enough for clarity

`planet_actor_list = list(map(init_planet, planet_filenames))`